### PR TITLE
HTTP errors from OpenCage API

### LINKED ISSUE #69: return status message from API on errors
To not only return error http code, but also a definition to go with it https://github.com/sckott/httpcode

### PR DESCRIPTION
**Pull request by dpprdan**
This fixes #69, i.e. HTTP error message from the OpenCage API are not returned to the user instead of the canned responses that were used before. With this, you get the following output:

``` r
library(opencage)

# 401 invalid key
oc_forward("Y", key = "thisisaninvalidkey")
#> Error: HTTP failure: 401
#> invalid API key
# 402 quota exceeded
oc_forward("Y", key = "4372eff77b8343cebfc843eb4da4ddc4")
#> Error: HTTP failure: 402
#> quota exceeded
# 403 blocked key
oc_forward("Y", key = "2e10e5e828262eb243ec0b54681d699a")
#> Error: HTTP failure: 403
#> suspended
```

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The new features include:
 ## Minor changes
 
 * The column name for both `languagecodes` and `countrycodes` is now `code`, and not `alpha2` and `Code`, respectively. 
+* HTTP error messages are returned directly from the API, not as a canned response from `code_message`. The latter has therefore been deleted.
```
Comment (maelle) [inline NEWS.md]: maybe add a tiny bit of context i.e. the messages from the API are more informative?

**Review by maelle — APPROVED** [review]
Looks great, I just added a very small comment. Thanks a ton!

File: NEWS.md (hunk 1 above)
Comment (dpprdan) [reply NEWS.md]: The messages are shorter actually: https://github.com/ropensci/opencage/issues/69#issuecomment-491495035
See e.g. the 403 status message above.

**Comment by dpprdan** [comment]
@maelle I added a few more details to the NEWS entry. Is that in the spirit you had in mind?

**Comment by maelle** [comment]
Yes, fantastic, thank you! Better for the changelog, and it'll also make our lives easier when we write a release blog post. :smile_cat: Feel free to merge :-)

**Pull request merged**